GUI/Controls/PlayTime.Designer.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace CKAN
+{
+    partial class PlayTime
```

Comment:
I'm not sure that users will want to open a separate screen to see just this information. What do you think of adding it as a column in the Manage Game Instances dialog instead? And possibly the title bar or status bar for the active game instance?

![image](https://user-images.githubusercontent.com/1559108/157084687-2defa337-08e7-4bf8-b2b9-44c055cc3a59.png)